docs/content/direct/get-started.md

```diff
@@ -25,9 +25,16 @@ Be sure to [install the software prerequisites](pre-reqs.md) _before_ running th
 The script will check for the pre-reqs and exit if they are not present.
 
 ### Run the script!
+The script can install KubeStellar's demonstration environment on top of kind or k3d
 
+For use with kind
 ```shell
-bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/tags/v{{ config.ks_latest_release }}/scripts/create-kubestellar-demo-env.sh)
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/tags/v{{ config.ks_latest_release }}/scripts/create-kubestellar-demo-env.sh) --platform kind
+```
+
+For use with k3d
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/refs/tags/v{{ config.ks_latest_release }}/scripts/create-kubestellar-demo-env.sh) --platform k3d
```

Comment:
when it is approved then we can take that temp link out

Reply:
Approval and even merging will not fix the problem, which is instructions telling the reader to use a URL that is 404. The problem will go away after this PR is merged and then a release is made.

This is not a blocker, just an FYI. If you wanted to fix it so that someone could succeed in the meantime, I think that it could be done as follows.

1. Make this PR say to use a URL that points into the `main` branch rather than the latest release.
2. Update the steps in`docs/content/direct/release.md` for making the next release to include (a) updating the URL here to refer to the latest release and (b) removing this step from that list of steps.

Reply:
It looks like you are planning the three-step approach that I suggested.

Reply:
Now it looks like we are back to the original problem. I see `{{ config.ks_latest_release }}` in `get-started.md`, which means that readers of `main` will get failures until the next release. Is this what you intend?

Reply:
I updated the getting started for the interim step outlined here.